sdk/golang/ccnp/quote/quote.go

```diff
@@ -0,0 +1,191 @@
+/*
+* Copyright (c) 2023, Intel Corporation. All rights reserved.<BR>
+* SPDX-License-Identifier: Apache-2.0
+ */
+
+package quote
+
+import (
+	"bytes"
+	"context"
+	"encoding/base64"
+	"encoding/binary"
+	"log"
+	"strings"
+	"time"
+
+	pb "github.com/intel/confidential-cloud-native-primitives/sdk/golang/ccnp/quote/proto"
+	pkgerrors "github.com/pkg/errors"
+	"google.golang.org/grpc"
+)
+
+const (
+	UDS_PATH = "unix:/run/ccnp/uds/quote-server.sock"
+	TYPE_TDX = "TDX"
+	TYPE_TPM = "TPM"
+)
+
+type TPMQuote struct{}
+
+type TDXQuote struct {
```

Comment:
Can we define sub structs such as TEE_TCB_INFO, TDINFO that can be reused in both TDXQuote and TDReport? Please take the tdx 1.0 spec as reference

Reply:
As discussed offline, we need more design on this and add a issue to follow this up https://github.com/intel/confidential-cloud-native-primitives/issues/137